### PR TITLE
#46 Creates an actual readme for the project.

### PR DESCRIPTION
**Pull request by Zonr0**
Issues:
* Text is a little barebones at the moment.
* Planned features section is blank
* Copyright only lists A Legitimate Buisness and not names, this may or
may not want to be changed to actual names.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,38 @@
-[![Stories in Ready](https://badge.waffle.io/legitbiz/SealsOfFate.svg?label=ready&title=Ready)](http://waffle.io/legitbiz/SealsOfFate)
+Project-Wide Issues on Waffle: [![Stories in Ready](https://badge.waffle.io/legitbiz/SealsOfFate.svg?label=ready&title=Ready)](http://waffle.io/legitbiz/SealsOfFate)
 
-# SealsOfFate
-A fork of the general SealsOfFate project for personal work, currently focusing on AI development. Like the main project, it is licensed under the MIT license, see LICENSE for details.
+# Seals Of Fate: Hats of Destiny; Icebergs of Doom
 
-###Week 3 Report###
-Code progress has been slower than expected over these last three weeks. Getting Unity and Git to interface has been unexpectedly difficult, leading to delays in setting up the project basics. Thus, the majority of the time has been spent doing research and design. In addition to the standard Unity tutorials, I've found a fantastic resource in "Programming Game AI by Example". It has been an excellent and practical guide that directly covers several of my goals.
+Copyright © 2017 A Legitimate Buisness
```
Comment (peschkaj) [inline README.md]: Legally, this should be "A Legitimate Business, LLC"

File: README.md [hunk 2]
```diff
@@ -1,11 +1,38 @@
...
 
-My initial prototype will include a single melee enemy. This enemy will only be active when the player is in the room, and when activated will charge at the player and attack, pathing aroudn obstacles. Once that initial protoype is finished, a secondary prototype is to implement a ranged enemy that attempts to maintain a minimum range from the player. These enemies will e implemented using a simple state machine with states for asleep, alert, seek, attack, and flee (for the ranged enemy). Pathfinding will be handled by A*.
+Seals of Fate is a prototype rogue-like built in the Unity engine using C#. 
+
+## Setup instructions
+
+1. The game is being developed in the latest version of the Unity game engine and is still in active development. As such, version 5.6 or later of the Unity editor is required to be installed to open and run the project.
```
Comment (peschkaj) [inline README.md]: "The game is being actively developed in the latest version of the Unity game engine . Unity v5.6 or later is required to open and run the project."

File: README.md [hunk 3]
```diff
@@ -1,11 +1,38 @@
-[![Stories in Ready](https://badge.waffle.io/legitbiz/SealsOfFate.svg?label=ready&title=Ready)](http://waffle.io/legitbiz/SealsOfFate)
+Project-Wide Issues on Waffle: [![Stories in Ready](https://badge.waffle.io/legitbiz/SealsOfFate.svg?label=ready&title=Ready)](http://waffle.io/legitbiz/SealsOfFate)
 
-# SealsOfFate
-A fork of the general SealsOfFate project for personal work, currently focusing on AI development. Like the main project, it is licensed under the MIT license, see LICENSE for details.
+# Seals Of Fate: Hats of Destiny; Icebergs of Doom
 
-###Week 3 Report###
-Code progress has been slower than expected over these last three weeks. Getting Unity and Git to interface has been unexpectedly difficult, leading to delays in setting up the project basics. Thus, the majority of the time has been spent doing research and design. In addition to the standard Unity tutorials, I've found a fantastic resource in "Programming Game AI by Example". It has been an excellent and practical guide that directly covers several of my goals.
+Copyright © 2017 A Legitimate Buisness, LLC
```
Comment (connorkuehl) [inline README.md]: Business*

Comment (connorkuehl) [inline README.md]: Also I think LLC's get a `.` at the end.

File: README.md [hunk 4]
```diff
@@ -1,11 +1,38 @@
...
 
-My initial prototype will include a single melee enemy. This enemy will only be active when the player is in the room, and when activated will charge at the player and attack, pathing aroudn obstacles. Once that initial protoype is finished, a secondary prototype is to implement a ranged enemy that attempts to maintain a minimum range from the player. These enemies will e implemented using a simple state machine with states for asleep, alert, seek, attack, and flee (for the ranged enemy). Pathfinding will be handled by A*.
+Seals of Fate is a prototype rogue-like built in the Unity engine using C#. 
+
+## Setup instructions
+
+1. The game is being actively developed in the latest version of the Unity game engine . Unity v5.6 or later is required to open and run the project.
```
Comment (connorkuehl) [inline README.md]: Maybe add a disclaimer saying that for now, the Unity client is required. Once we've reached certain milestones we will publish builds for specific platforms.

**Pull request merged**